<launch>
  <arg name="maxDisappeared"         default="10" />
  <arg name="path_integ"         default="0.1" />
  <arg name="effect_x_range"         default="0.115" />
  <arg name="effect_y_range"         default="0.15" />
  <arg name="start_water_range"         default="0.15" />
  <arg name="finish_water_range"         default="0.07" />
  <arg name="path_x_max"         default="0.2" 
  />
  <arg name="set_valve_pwm"         default="100"/>
  <arg name="watering_mode"         default="5" />  
  <arg name="bias"         default="3" /> 

  <!-- wapabawama_ros -->
  <group ns="wapabawama_ros">
    <include file="$(find wapabawama_ros)/launch/move.launch"/>
    
    <!-- <node name="watering_vl1" pkg="wapabawama_ros" type="watering" output="screen">
      <param name="path_name"        value="la1/path"/>
      <param name="frame_name" value="la1"/>
      <param name="watering_mode" value="$(arg watering_mode)"/>
      <rosparam param="amount_list">[0,4.584,17.2,31.134,43.058,55.714,68.522,74.798,83.774,93.814,103.11]
      </rosparam>
      <param name="pwm_name"         value="valve1/pwm"/>
      <param name="stop_sec"         value="2"/>
      <param name="bias"         value="0.04"/>   
      <param name="center_x"         value="0.01"/>
      <param name="set_pwm"            value="$(arg set_valve_pwm)"/>
      <param name="start_water_range" value="$(arg start_water_range)"/>
      <param name="finish_water_range" value="$(arg finish_water_range)"/>
    </node> -->
    <!-- <node name="watering_vl2" pkg="wapabawama_ros" type="watering" output="screen">
      <param name="path_name"        value="la2/path"/>
      <param name="frame_name" value="la2"/>
      <param name="watering_mode" value="$(arg watering_mode)"/>
      <rosparam param="amount_list">[0,10,22.8,35.8,48,62.6,72.2,83,92,99.6,106.6]</rosparam>
      <param name="pwm_name"         value="valve2/pwm"/>
      <param name="stop_sec"         value="2"/>
      <param name="bias"         value="0.06"/>   
      <param name="center_x"         value="0.25"/>
      <param name="set_pwm"            value="$(arg set_valve_pwm)"/>
      <param name="start_water_range" value="$(arg start_water_range)"/>
      <param name="finish_water_range" value="$(arg finish_water_range)"/> 
    </node> -->
    <!-- <node name="watering_vl3" pkg="wapabawama_ros" type="watering" output="screen">
      <param name="path_name"        value="la3/path"/>
      <param name="frame_name" value="la3"/>
      <param name="watering_mode" value="$(arg watering_mode)"/>
      <param name="stop_sec"         value="2"/>
      <param name="bias"         value="0.04"/>   
      <param name="pwm_name"         value="valve3/pwm"/>
      <param name="center_x"         value="-0.22"/>
      <rosparam param="amount_list">[0,1.184,12.71,26.124,36.546,49.296,59.41,70.952,79.852,89.22,95.444]</rosparam>
      <param name="set_pwm"            value="$(arg set_valve_pwm)"/>    
      <param name="start_water_range" value="$(arg start_water_range)"/>
      <param name="finish_water_range" value="$(arg finish_water_range)"/>
    </node>     -->
<!-- 
    <node name="record_1" pkg="wapabawama_ros" type="record_place">
      <param name="record_name"        value="la1/odem"/>
      <param name="frame_name"         value="la1"/>
    </node>
    <node name="record_2" pkg="wapabawama_ros" type="record_place">
      <param name="record_name"        value="la2/odem"/>
      <param name="frame_name"         value="la2"/>
    </node>
    <node name="record_3" pkg="wapabawama_ros" type="record_place">
      <param name="record_name"        value="la3/odem"/>
      <param name="frame_name"         value="la3"/>
    </node> -->

  </group>

  <!-- <node name="rviz" pkg="rviz" type="rviz" respawn="false" args="-d $(find wapabawama_ros)/resources/rviz2.rviz" /> -->
</launch>
